extensions/markdown-math/package.json

```diff
@@ -58,10 +58,10 @@
     ],
     "notebookRenderer": [
       {
-        "id": "markdownItRenderer-katex",
+        "id": "vscode.markdown-it-katex-extension",
```

Comment:
This change doesn't really matter. Just adding the `vscode.` prefix to be consistent